Automattic/wp-calypso
# Jetpack: update Backup success image

#### Changes proposed in this Pull Request

* Update the Backup success image to the standard Material icon `cloud_done_24px` instead of using a custom icon.

#### Testing instructions

* Fire up this PR.
* Go to Calypso Green: http://jetpack.cloud.localhost:3000/
* Select a site with a Backup plan, and previous successful backups.
* Ensure you see the new icon.

#### Screenshots

Before | After
--|--
![image](https://user-images.githubusercontent.com/390760/112147791-026e2580-8bd5-11eb-80d1-b960a2d223df.png) | ![image](https://user-images.githubusercontent.com/390760/112147800-0437e900-8bd5-11eb-96c9-79555eb9c7f5.png)
